spec/integration/integration_helper.rb

```diff
@@ -37,7 +37,7 @@ def connect!(port)
   def ar_conn(port)
     ActiveRecord::Base.establish_connection(
       :adapter  => defined?(Mysql2) ? 'mysql2' : 'mysql',
-      :host     => '127.0.0.1',
+      :host     => 'localhost',
```

Comment:
this is just for the integration tests.  it's so that they actually pass :panda_face: